ecosystem/python/sdk/aptos_sdk/metadata.py

```diff
@@ -0,0 +1,11 @@
+import importlib.metadata as metadata
+
+# constants
+package_name = "aptos-sdk"
```

Comment:
nit: PACKAGE_NAME

Reply:
Actually I made a dumb mistake, I should have a method that get the whole header_value instead of just version. Adding that in right now